cms/templates/js/course-outline.underscore

```diff
@@ -57,7 +60,16 @@ if (staffOnlyMessage) {
         addStatusMessage(messageType, messageText);
     }
 
-    if (hasPartitionGroups) {
+    if (selectedPartitionIndex !== -1 && !isNaN(selectedPartitionIndex) && xblockInfo.isVertical()) {
```

Comment:
My understanding is that user partition IDs shouldn't ever be below 0, the !isNaN is used mostly as a conditional check that the selectedPartitionIndex exists and wasn't parsed inappropriately, ex. ```
> !isNaN({})
false
> !isNaN(undefined)
false
> !isNaN(parseInt('a'))
false
```